testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/carbon.in

```diff
@@ -1,19 +1,116 @@
-# This file holds atmospheric CO2 concentrations in PPM per scenario.
-# The data is organized by year, with each year having a concentration value.
+#------ Input file for yearly atmospheric CO2 concentration

```

Comment:
All changes to testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/carbon.in need to be reverted. STEPWAT2 uses simulation time and not calendar years as SOILWAT2 does